olive/evaluator/olive_evaluator.py

```diff
@@ -237,9 +264,13 @@ def _evaluate_onnx_accuracy(
     def _evaluate_onnx_latency(
         self, model: OliveModel, metric: Metric, dataloader: Dataset, device: Device = Device.CPU, post_func=None
     ) -> Dict[str, Any]:
-        warmup_num = metric.metric_config.warmup_num
-        repeat_test_num = metric.metric_config.repeat_test_num
-        sleep_num = metric.metric_config.sleep_num
+        warmup_num, repeat_test_num, sleep_num = None, None, None
```

Comment:
lot of redundancy code. Could we remove them by a method or function?

Reply:
Updated.